src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tr = tbody.rows;
+let sortNames;
+
+thead.addEventListener('click', (e) => {
+  switch (e.target.textContent) {
+    case 'Name':
+      sortNames = function(arr) {
+        return [...arr].sort((a, b) =>
+          a.children[0].textContent < b.children[0].textContent ? -1 : 1);
+      };
+
+      tbody.append(...sortNames(tr));
+      break;
+
+    case 'Position':
+      sortNames = function(arr) {
+        return [...arr].sort((a, b) =>
+          a.children[1].textContent < b.children[1].textContent ? -1 : 1);
+      };
+
+      tbody.append(...sortNames(tr));
+      break;
```

Comment:
combine it

Reply:
how to combine if I have different array elements?  
children[0] and children[1]
help please

Reply:
you can create 2 different functions for sorting text and number values: `sortText` and `sortNumbers`.
and then you can just reuse them.
let's consider the example:
```
const sortText = (data, childIdx) => data
  .sort((a, b) => a.children[childIdx].textContent < b.children[childIdx].textContent ? -1 : 1);
```
then you will just reuse the function as `sortText([...arr], 0)`

Reply:
Hi!
You can create common function like this:
```
const sortByIndex = (items, index) => [...items].sort(a, b) => 
  a.children[index].textContent < b.children[index].textContent ? -1 : 1;
```
And use it for all cases:
```
case 'Name':
  tbody.append(sortByIndex(tr, 0));
  break;

case 'Position': 
  tbody.append(sortByIndex(tr, 0));
  break;

...
```

Reply:
> Check my comment, please.

Thank